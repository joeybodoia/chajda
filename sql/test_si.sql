CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."');
