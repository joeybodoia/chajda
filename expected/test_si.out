CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                     spacy_lemmatize                                                      
--------------------------------------------------------------------------------------------------------------------------
   පරීක්ෂණ:2 අවස්ථා:3 උත්පාදනය:4 කිරීම:5 සඳහා:6 මම:7 ගූගල්:8 පරිවර්තනයට:9 ඇතුළත්:10 කරන:11 මගේ:12 උදාහරණ:13 පරීක්ෂණ:14 වාක්යය:15 මෙයයි:16
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්යය මෙයයි
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                     spacy_lemmatize                                                      
--------------------------------------------------------------------------------------------------------------------------
   පරීක්ෂණ:2 අවස්ථා:3 උත්පාදනය:4 කිරීම:5 සඳහා:6 මම:7 ගූගල්:8 පරිවර්තනයට:9 ඇතුළත්:10 කරන:11 මගේ:12 උදාහරණ:13 පරීක්ෂණ:14 වාක්යය:15 මෙයයි:16
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්යය මෙයයි
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   ":2 පරීක්ෂණ:3 අවස්ථා:4 උත්පාදනය:5 කිරීම:6 සඳහා:7 මම:8 ගූගල්:9 පරිවර්තනයට:10 ඇතුළත්:11 කරන:12 මගේ:13 උදාහරණ:14 පරීක්ෂණ:15 වාක්‍යය:16 මෙයයි:17 .:18 ":19
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි . "
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   ":2 පරීක්ෂණ:3 අවස්ථා:4 උත්පාදනය:5 කිරීම:6 සඳහා:7 මම:8 ගූගල්:9 පරිවර්තනයට:10 ඇතුළත්:11 කරන:12 මගේ:13 උදාහරණ:14 පරීක්ෂණ:15 වාක්‍යය:16 මෙයයි:17 .:18 ":19
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි . "
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                     spacy_lemmatize                                                      
--------------------------------------------------------------------------------------------------------------------------
   පරීක්ෂණ:2 අවස්ථා:3 උත්පාදනය:4 කිරීම:5 සඳහා:6 මම:7 ගූගල්:8 පරිවර්තනයට:9 ඇතුළත්:10 කරන:11 මගේ:12 උදාහරණ:13 පරීක්ෂණ:14 වාක්යය:15 මෙයයි:16
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්යය මෙයයි
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                     spacy_lemmatize                                                      
--------------------------------------------------------------------------------------------------------------------------
   පරීක්ෂණ:2 අවස්ථා:3 උත්පාදනය:4 කිරීම:5 සඳහා:6 මම:7 ගූගල්:8 පරිවර්තනයට:9 ඇතුළත්:10 කරන:11 මගේ:12 උදාහරණ:13 පරීක්ෂණ:14 වාක්යය:15 මෙයයි:16
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්යය මෙයයි
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   ":2 පරීක්ෂණ:3 අවස්ථා:4 උත්පාදනය:5 කිරීම:6 සඳහා:7 මම:8 ගූගල්:9 පරිවර්තනයට:10 ඇතුළත්:11 කරන:12 මගේ:13 උදාහරණ:14 පරීක්ෂණ:15 වාක්‍යය:16 මෙයයි:17 .:18 ":19
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි . "
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   ":2 පරීක්ෂණ:3 අවස්ථා:4 උත්පාදනය:5 කිරීම:6 සඳහා:7 මම:8 ගූගල්:9 පරිවර්තනයට:10 ඇතුළත්:11 කරන:12 මගේ:13 උදාහරණ:14 පරීක්ෂණ:15 වාක්‍යය:16 මෙයයි:17 .:18 ":19
(1 row)

SELECT spacy_lemmatize('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි . "
(1 row)

SELECT spacy_tsvector('si',' "පරීක්ෂණ අවස්ථා උත්පාදනය කිරීම සඳහා මම ගූගල් පරිවර්තනයට ඇතුළත් කරන මගේ උදාහරණ පරීක්ෂණ වාක්‍යය මෙයයි."');
                                                                spacy_tsvector                                                                
----------------------------------------------------------------------------------------------------------------------------------------------
 'අවස්ථා':3 'ඇතුළත්':10 'උත්පාදනය':4 'උදාහරණ':13 'කරන':11 'කිරීම':5 'ගූගල්':8 'පරිවර්තනයට':9 'පරීක්ෂණ':2,14 'මගේ':12 'මම':7 'මෙයයි':16 'වාක්යය':15 'සඳහා':6
(1 row)

